flax/training/train_state.py

```diff
@@ -22,29 +22,30 @@
 class TrainState(struct.PyTreeNode):
   """Simple train state for the common case with a single Optax optimizer.
 
-  Synopsis:
+  Synopsis::
 
-    state = TrainState.create(
-        apply_fn=model.apply,
-        params=variables['params'],
-        tx=tx)
-    grad_fn = jax.grad(make_loss_fn(state.apply_fn))
-    for batch in data:
-      grads = grad_fn(state.params, batch)
-      state = state.apply_gradients(grads=grads)
+      state = TrainState.create(
+          apply_fn=model.apply,
+          params=variables['params'],
+          tx=tx)
+      grad_fn = jax.grad(make_loss_fn(state.apply_fn))
+      for batch in data:
+        grads = grad_fn(state.params, batch)
+        state = state.apply_gradients(grads=grads)
```

Comment:
I don't think you need 4 indents, 2 indents and the double colon with an empty newline should be sufficient to get code view.

Reply:
Correct. Updated